docs/conf.py

```diff
@@ -0,0 +1,67 @@
+# Configuration file for the Sphinx documentation builder.
```

Comment:
What do you think if we format the `.py` files in `docs` with
```bash
tox -e format -- docs
```
?

Then it'll match our usual code style.